include/flux/core/utils.hpp

```diff
@@ -70,6 +68,36 @@ FLUX_EXPORT
 template <std::integral To>
 inline constexpr auto checked_cast = detail::checked_cast_fn<To>{};
 
+namespace detail {
+template <bool FirstValue, class First, class... Rest>
+struct disjunction_t { // handle true trait or last trait
+    using type = First;
+};
+
+
+template <class False, class Next, class... Rest>
+struct disjunction_t<false, False, Next,
+    Rest...> { // first trait is false, try the next trait
+    using type = typename disjunction_t<Next::value, Next, Rest...>::type;
+};
+
+template <class... Traits>
+struct disjunction : std::false_type {}; // If _Traits is empty, false_type
+
+template <class First, class... Rest>
+struct disjunction<First, Rest...>
+    : disjunction_t<First::value, First, Rest...>::type {
+    // the first true trait in _Traits, or the last trait if none are true
+};
+
+template <class... Traits>
+inline constexpr bool disjunction_v = disjunction<Traits...>::value;
+
+// VARIABLE TEMPLATE _Is_any_of_v
+template <class T, class... Types>
+inline constexpr bool is_any_of = disjunction_v<std::is_same<T, Types>...>;
+;
+} // namespace detail
```

Comment:
FYI, there's a `detail::any_of` concept in `split_string.hpp` that is a bit simpler than the C++17 approach :). You can move it to utils.hpp if you like. 

https://github.com/tcbrindle/flux/blob/main/include/flux/op/split_string.hpp#L17-L18

Reply:
(Not a full review as I can see you're still working on it, I just happened to notice this :) )

Reply:
Oh thank you! That's very helpful :)